packages/@aws-cdk/lambda-layer-awscli/lib/awscli-layer.ts

```diff
@@ -1,19 +1,150 @@
+/* eslint-disable no-console */
+import * as childproc from 'child_process';
+import * as fs from 'fs';
 import * as path from 'path';
 import * as lambda from '@aws-cdk/aws-lambda';
-import { FileSystem } from '@aws-cdk/core';
+import { Annotations, FileSystem } from '@aws-cdk/core';
+import { debugModeEnabled } from '@aws-cdk/core/lib/debug';
+import * as cxapi from '@aws-cdk/cx-api';
 import { Construct } from 'constructs';
 
 /**
  * An AWS Lambda layer that includes the AWS CLI.
  */
 export class AwsCliLayer extends lambda.LayerVersion {
+  /**
+   * @internal
+   */
+  public static _tryLoadPackage(targetVersion: string, logs: string[]): any {
```

Comment:
If this is public/`@internal` in order to be able to test it, rather just stick it in a separate unexported file (`lib/private/dynload.ts` or whatever) and import  that file directly in testing.

Reply:
Ah yep, that's way better. Thanks!